app/models/casa_org.rb

```diff
@@ -23,6 +23,10 @@ def volunteers
     Volunteer.in_organization(self)
   end
 
+  def cases
+    CasaCase.where(casa_org_id: id)
```

Comment:
Stylistically, it could be better if this were an activerecord relationship with a `has_many`

Reply:
Looks like it already had a has_many relationship and I just missed it.